src/main/java/uk/gov/digital/ho/hocs/queue/migration/MigrationService.java

```diff
@@ -28,8 +37,17 @@ public MigrationService(WorkflowClient workflowClient,
 
     public void createMigrationCase(MigrationData migrationCaseData, MigrationCaseTypeData migrationCaseTypeData) {
 
-        log.info("create Migration Case, started : type {}", migrationCaseData.getComplaintType());
+        // dummy documents list
+        DocumentSummary documentSummary = new DocumentSummary("migration","","");
+        var migrationRequest = composeMigrateCaseRequest(migrationCaseData, migrationCaseTypeData, documentSummary);
+        CreateCaseworkCaseResponse caseResponse = caseworkClient.migrateCase(migrationRequest);
+        log.info("Created migration case {}", caseResponse.getUuid());
+    }
+
+    private CreateMigrationCaseRequest composeMigrateCaseRequest(MigrationData migrationData, MigrationCaseTypeData migrationCaseTypeData, DocumentSummary documentSummary) {
+        Map<String, String> initialData = Map.of(CHANNEL_LABEL, migrationCaseTypeData.getOrigin());
 
+        return new CreateMigrationCaseRequest(migrationData.getComplaintType(), migrationData.getDateReceived(), List.of(documentSummary), initialData, "MIGRATION");
```

Comment:
`MIGRATION` is this temporary? I presume it is as there will have to be a unique stage type per case type.